### PR TITLE
Setup Create React up

### PR DESCRIPTION
**Pull request by milen-ale**
- install nod.js.
- Setup Create Up.

### CODE REVIEW

**Review by vechicin — COMMENTED** [review]
# Hi @milen-ale ✋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😞 

## Invalid Code Review Request ⚠️ 

You have submitted a project, that doesn't have linters set up.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by vechicin — COMMENTED** [review]
# Hi @milen-ale ✋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😞 

## Invalid Code Review Request ⚠️ 

You have submitted a project, that has no linters set up. Please follow [THIS LINK](https://github.com/microverseinc/linters-config/blob/master/react-redux/README.md) to properly set them up.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:

- [ ] You didn't document your work professionally:
   - You forgot to edit your README file.

image

- [ ] Gitflow is not correctly used.

image

- [ ] Linters are not configured or working for this project.

So please fix those issues and ask for another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by vechicin — COMMENTED** [review]
# Hi @milen-ale ✋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😞 

## Invalid Code Review Request ⚠️ 

You have submitted a project, that linters are not correctly setup and README file hasn't been edited properly.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @milen-ale , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :clap: 
:ballot_box_with_check: App has correct functionality :+1: 
:ballot_box_with_check: Linters are passing :ok_hand: 
:ballot_box_with_check: Good commit history :clap: 



## Required Changes ♻️
- [ ] Kindly edit your README file using this [Microverse template](https://github.com/microverseinc/readme-template) with your information and information about how to set up and run your project locally :100: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,10 +1,66 @@
-# Math-Magicians
+# React and Redux
 
+If you are not familiar with linters and GitHub Actions, read [root level README](../README.md).
 
+## Set-up GitHub Actions
 
+This GitHub Action is going to run [ESLint](https://eslint.org/) and [Stylelint](https://stylelint.io/) to help you find style issues.
```
Comment (Kingstalux) [inline README.md]: - [ ] Good job so far :+1: but your README file is not yet professional. Kindly use this [Microverse template](https://github.com/microverseinc/readme-template), edit it with your info and info about your project, and also add steps a user would need to set up and run your project locally :100:

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @milen-ale , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kingstalux — APPROVED** [review]
Hi @milen-ale,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966
 

Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**